### ISSUE TITLE
.hpp and .cpp files not showing up (solution generated by FASTbuild)

### ISSUE THREAD

**Issue by uucidl** [slynch8/10x#308]
In a large project (when opening the folder directly, 10x reaches 8GB of RAM) we use FASTbuild to build.

When I load our .sln generated with FASTbuild, 10x does not "see" the .hpp and .cpp files.

FASTbuild generates .vcxproj files with this kind of structure: 
```
 <ItemGroup>
    <CustomBuild Include="..\..\..\Code\Core\Containers\Array.h" />
    <CustomBuild Include="..\..\..\Code\Core\Env\Assert.cpp" />
```
see https://github.com/fastbuild/fastbuild/blob/84c8f2506787f57d582117b825036eedc5035621/Code/Tools/FBuild/FBuildTest/Data/TestProjectGeneration/Solution_ExternalProject/ExternalMissingProjectGUID.vcxproj

Is it an issue caused by these `CustomBuild` tags? Who is risk/wrong here? These solutions do show the files in Visual Studio.

**Comment by slynch8** [comment]
Looks like 10x isn't recognising "CustomBuild" as a valid file element. I'll get that added...

**Comment by uucidl** [comment]
It seems fixed in 0.27.6.0 that I just updated to?

**Comment by slynch8** [comment]
yes, was just about to reply :-) Should be fixed now.
